minirt/src/minirt.c

```diff
@@ -16,30 +16,33 @@
 #include <fcntl.h>
 #include "libft/libft.h"
 
+int Check_key(t_list *lines);
 int ft_isspace(char c);
-t_config_map getMapFromLine(char *line) {
+t_config_map *getMapFromLine(char *line)
+{
 	int i = 0;
 	int j = 0;
 	int k = 0;
 	char *key = NULL;
 	char *value = NULL;
-	t_config_map map;
+	t_config_map *map = malloc(sizeof (t_config_map));
+
 	while (i < 5)
-		map.value[i++] = NULL;
+		map->value[i++] = NULL;
 	i = 0;
 	while (line[i] && ft_isalpha(line[++i]));
```

Comment:
wrong check if I put R@ it will take only R for the key
we should check if there is a space